Godeps/_workspace/src/github.com/Clever/pathio/Makefile

```diff
@@ -0,0 +1,29 @@
+SHELL := /bin/bash
+PKG = github.com/Clever/pathio
+PKGS := $(PKG)
+READMES = $(addsuffix /README.md, $(PKGS))
+
+.PHONY: test golint README
```

Comment:
`$(PKGS)` is also phony

`README` is not a target

`golint` can be made un-phony if you wish. See the approach in the stoked Makefile